tests/src/Unit/StreamContextFactoryTest.php

```diff
@@ -47,4 +47,56 @@ public function testProxyStreamContext()
         $proxyAuthHeader = 'Proxy-Authorization: Basic ' . base64_encode($proxy['login'] . ':' . $proxy['password']);
         $this->assertContains($proxyAuthHeader, $options['http']['header']);
     }
+
+    public function testAuthorizationHeaderShouldBeDefined()
```

Comment:
The method naming pretty much says what is tested here but I would appreciate a DocBlock for the test anyway for the sake of consistency with the test of the test suite.